src/UCTSearch.cpp

```diff
@@ -522,12 +521,18 @@ int UCTSearch::get_best_move(passflag_t passflag) {
     return bestmove;
 }
 
-std::string UCTSearch::get_pv(FastState & state, UCTNode& parent) {
+std::string UCTSearch::get_pv(GameState & state, UCTNode& parent) {
     if (!parent.has_children()) {
```

Comment:
This should be a check for EXPANDED instead, because we can't safely traverse partially expanded nodes while the search is still going on.